recipes/superdsm/run_test.sh

```diff
@@ -0,0 +1,6 @@
+set -e
+set -x
+
+echo "Running SuperDSM test suite"
+
+python -m unittest -v tests
```

Comment:
In general, we do not have the resources available to run unit tests on our infrastructure (see https://bioconda.github.io/contributor/guidelines.html#tests), so please simplify this. Typically for Python packages the test is a simple `import` statement just to make sure it's installed (see recent Python recipes for examples).

Reply:
I have reduced the testing routine in https://github.com/bioconda/bioconda-recipes/pull/40220/commits/e8590f8068ebee9539079075e7e595a07ef4e666. It should be fine now.